src/index.tsx

```diff
@@ -19,6 +19,13 @@ import mergeExtraData from './merge-extradata';
 
 const isAndroidRTL = I18nManager.isRTL && Platform.OS === 'android';
 
+type OrientationType = "horizontal" | "vertical"
+
+const Orientations = {
+  horizontal: "horizontal",
+  vertical: "vertical"
+}
```

Comment:
```suggestion
enum Orientation {
  HORIZONTAL,
  VERTICAL
}
```
I think it would be more appropriate to use an enum here

Reply:
ok i'll change it and let you know

Reply:
done and commited